src/core_plugins/console/public/src/input.js

```diff
@@ -16,7 +16,11 @@ let input;
 export function initializeInput($el, $actionsEl, $copyAsCurlEl, output) {
   input = new SenseEditor($el);
 
-  uiModules.get('app/sense').setupResizeCheckerForRootEditors($el, input, output);
+  // this may not exist if running from tests
+  let appSense = uiModules.get('app/sense');
+  if (appSense.setupResizeCheckerForRootEditors) {
```

Comment:
Can we mock this in the tests rather than altering the code to accommodate the tests?